templates/components/api-server/template/server/server.js

```diff
@@ -3,20 +3,23 @@ var boot = require('loopback-boot');
 
 var app = module.exports = loopback();
 
+// boot scripts mount components like REST API
+boot(app, __dirname);
+
 app.start = function() {
   // start the web server
   return app.listen(function() {
     app.emit('started');
-    console.log('Web server listening at: %s', app.get('url'));
+    var baseUrl = app.get('url').replace(/\/$/, '');
+    console.log('Web server listening at: %s', baseUrl);
+    if (app.get('loopback-explorer')) {
+      var explorerPath = app.get('loopback-explorer').mountPath;
+      console.log('Browse your REST API at %s%s', baseUrl, explorerPath);
+    }
   });
 };
 
-// Bootstrap the application, configure models, datasources and middleware.
-// Sub-apps like REST API are mounted via boot scripts.
-boot(app, __dirname, function(err) {
-  if (err) throw err;
-
-  // start the server if `$ node server.js`
-  if (require.main === module)
-    app.start();
-});
```

Comment:
Please revert this change, as it is effectively removing support for async boot scripts.